docs_src/design/adr/0026-Common Configuration.md

```diff
@@ -0,0 +1,394 @@
+  # Reader-10-EF-25_1 = "Freezer"
+  # Reader-10-EF-25_2 = "Backroom"
+  [AppCustom.Aliases]
+
+  # See: https://github.com/edgexfoundry/app-rfid-llrp-inventory#configuration
+  [AppCustom.AppSettings]
+  DeviceServiceName = "device-rfid-llrp"
+  AdjustLastReadOnByOrigin = true
+  DepartedThresholdSeconds = 600
+  DepartedCheckIntervalSeconds = 30
+  AgeOutHours = 336
+  MobilityProfileThreshold = 6.0
+  MobilityProfileHoldoffMillis = 500.0
+  MobilityProfileSlope = -0.008
+```
+
+#### Device MQTT  Private Configuration
+
+```toml
+MaxEventSize = 0 # value 0 unlimit the maximum event size that can be sent to message bus or core-data
+
+[Writable]
+  # InsecureSecrets are required for when Redis is used for message bus
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.MQTT]
+    path = "credentials"
+      [Writable.InsecureSecrets.MQTT.Secrets]
+      username = ""
+      password = ""
+
+[Service]
+Port = 59982
+StartupMsg = "device mqtt started"
+
+[Clients]
+  [Clients.core-data]
+  Protocol = "http"
+  Host = "localhost"
+  Port = 59880
+
+  [Clients.core-metadata]
+  Protocol = "http"
+  Host = "localhost"
+  Port = 59881
+
+[MessageQueue]
+PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name>/<source-name> will be added to this Publish Topic prefix
+  [MessageQueue.Optional]
+  # Default MQTT & NATS Specific options that need to be here to enable environment variable overrides of them
+  ClientId = "device-mqtt"
+  [MessageQueue.Topics]
+  CommandRequestTopic = "edgex/device/command/request/device-mqtt/#"   # subscribing for inbound command requests
+  CommandResponseTopicPrefix = "edgex/device/command/response"   # publishing outbound command responses; <device-service>/<device-name>/<command-name>/<method> will be added to this publish topic prefix
+
+[MQTTBrokerInfo]
+Schema = "tcp"
+Host = "localhost"
+Port = 1883
+Qos = 0
+KeepAlive = 3600
+ClientId = "device-mqtt"
+
+CredentialsRetryTime = 120 # Seconds
+CredentialsRetryWait = 1 # Seconds
+ConnEstablishingRetry = 10
+ConnRetryWaitTime = 5
+
+# AuthMode is the MQTT broker authentication mechanism. Currently, "none" and "usernamepassword" is the only AuthMode supported by this service, and the secret keys are "username" and "password".
+AuthMode = "none"
+CredentialsPath = "credentials"
+
+# Comment out/remove when using multi-level topics
+IncomingTopic = "DataTopic"
+ResponseTopic = "ResponseTopic"
+UseTopicLevels = false
+
+# Uncomment to use multi-level topics
+# IncomingTopic = "incoming/data/#"
+# ResponseTopic = "command/response/#"
+# UseTopicLevels = true
+
+    [MQTTBrokerInfo.Writable]
+    # ResponseFetchInterval specifies the retry interval(milliseconds) to fetch the command response from the MQTT broker
+    ResponseFetchInterval = 500
+```
+
+### Modules and Services Impacted
+
+The following modules and services are impacted:
+

```

Comment:
Is not security-secrestore-setup needed to seed the Basic Auth secret for the new configuration service?

Reply:
added!